components/Projects/FiltersBar/FiltersBar.tsx

```diff
@@ -107,6 +110,7 @@ const FiltersBar: React.FC = () => {
                           ))}
                         </div>
                       </div>
+                      {/* sort by set 2 */}
                       <div className="flex gap-4 md:gap-[26px] justify-center md:justify-normal md:items-center">
```

Comment:
Same comment as above regarding the custom component

Reply:
I updated the comment.
There will be some benefit in making it as a standalone component but not much.